### PR TITLE
chore: update pubspec files

### PR DESCRIPTION
**Pull request by kevmoo**
Allow latest analyzer version
Add an upper constraint to the Dart SDK

### CODE REVIEW

**Comment by kevmoo** [comment]
@yjbanov?

File: modules/angular2/pubspec.yaml [hunk 1]
```diff
@@ -7,9 +7,9 @@ authors:
 description: Angular 2 for Dart - a web framework for modern web apps
 homepage: <%= packageJson.homepage %>
 environment:
-  sdk: '>=1.9.0-dev.8.0'
+  sdk: '>=1.9.0 <2.0.0'
```
Comment (yjbanov) [inline modules/angular2/pubspec.yaml]: Can you make it `>=1.10.0`? There's a bug in `1.9.0` that kills our constructor parameter annotations.

Comment (kevmoo) [inline modules/angular2/pubspec.yaml]: If the team is comfortable with that, sure.

Comment (kevmoo) [inline modules/angular2/pubspec.yaml]: Done

**Comment by kevmoo** [comment]
PTAL

**Comment by alexeagle** [comment]
broke the build, can you fix soon or revert?

**Comment by kevmoo** [comment]
https://github.com/angular/angular/pull/1724

**Pull request merged**